### PR TITLE
List toast

### PR DESCRIPTION
**Pull request by shashikantmaurya0007**
Screenshot (1528)
feature:added List and toast
Screenshot (1529)
Screenshot (1530)
Screenshot (1531)
Screenshot (1532)

### CODE REVIEW

File: List/list.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    background-color: var(--list-hover-bgColor);
+}
+
+.list-heading {
+    font-weight: bold;
+    height: 3rem;
+    display: flex;
+    align-items: center;
+    padding-left: 2rem;
+    border-bottom: 1px solid lightgray;
```
Comment (ravikumar0403) [inline List/list.css:48]: Use variables for colors

File: Toast/toast.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (ravikumar0403) [inline Toast/toast.html:8]: Update the title here to be relevant with the page

**Review by ravikumar0403 — APPROVED** [review]
Looks good, Left a couple of sanity comments

**Review by kotesh-arya — APPROVED** [review]
Looks good for a merge 👍🏻

File: List/list.html [hunk 3]
```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (yashpatel1797) [inline List/list.html:7]: You can add author tag.

File: Input/input.html [hunk 4]
```diff
@@ -140,9 +140,18 @@ <h1 class="content-heading">Checkbox</h1>
...
+                        style="
+                width: 97%;
+                height: 745px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              " sandbox="allow-scripts allow-same-origin">
```
Comment (yashpatel1797) [inline Input/input.html:152]: not mandatory but if you can give class here and write CSS in the file

**Review by yashpatel1797 — APPROVED** [review]
Looks good to me.

**Pull request merged**